api/v1/qdrantcluster_types.go

```diff
@@ -15,8 +15,16 @@ const (
 )
 
 const (
-	// The Annotation key to trigger a restart.
+	// The annotation key to trigger a restart.
+	// The value should be a [RFC3339 formatted] date.
+	// If the value is updated it will  retrigger the restart.
+	// For historical reasons the key doesn't start with `operator.qdrant.com/`
+	// The annotation should be placed on the QdrantCluster instance.
 	RestartedAtAnnotationKey = "restartedAt"
+	// The annotation key to recreate a certain node.
+	// The value should contain the node index (e.g. "3").
+	// The annotation should be placed on the QdrantCluster instance.
+	RecreateNodeAnnotationKey = "operator.qdrant.com/recreate-node"
```

Comment:
where is this used?

Reply:
This is to implement: https://github.com/qdrant/cloud-pm/issues/1376 and will be used in the next operator PR.

Reply:
> Did you check the ticket?

Yes, if I understand correctly, that annotation is going to be used only by the operator, and not by agent or cluster-api? If true, then we do not need to publish this constant here at all. Wdyt?

Reply:
It's part of the API (and we ned to document this for private cloud usage)

Reply:
PS: We can implement the 'replace' functionality on the admin dashboard  by adding a button (as we refactored it, so that's not only readOnly).